echopype/consolidate/api.py

```diff
@@ -168,7 +168,8 @@ def sel_interp(var):
             )
         else:
             # Interpolation. time1 is always associated with location data
-            return coord_var.interp(time1=ds["ping_time"])
+            # The first value ends up being always nan. Use "extrapolate"
+            return coord_var.interp(time1=ds["ping_time"], kwargs={"fill_value": "extrapolate"})
```

Comment:
I think this may be that the first `ping_time` precedes the first NMEA sentence time, so there's nothing to interpolate against. How about we keep it as NaN? That is technically more correct (since we do not know the exact ping location), and also likely avoid problems later on for computing distances between pings (for NASC computation).

For most of the ship survey extrapolating the first ping is fine since the ping rate is high, but if that's not the case the extrapolation error may be non-negligible.

Reply:
BTW, how about changing `coord_var` to something like `da_var`? When I was reading the code, I spent a few confused minutes trying to figure out why we can interpolate the coordinates of an xr data array. 😅

Reply:
Regarding `coord_var` vs `position_var`, I see your point. I'll make the change in my local clone and push it to the PR.

Reply:
In the latest commit I've removed extrapolation and renamed "coord" variables to "position" in both `consolidate/api.py` and the test.